examples/gcp-values-tls.yaml

```diff
@@ -4,9 +4,12 @@
 deployTarget: GOOGLE
 
 dash:
-  tls:
+  ingress:
     enabled: true
-    secretName: "pach-tls" #Will use same secret as pach
+    host: "blah.com"
```

Comment:
I think this ought to be under an [RFC 6761 special domain](https://datatracker.ietf.org/doc/html/rfc6761), specifically `blah.test`, `blah.example` or `blah.invalid`. `blah.com` is registered and belongs to someone.